client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/sidebar.tsx

```diff
@@ -10,11 +10,13 @@ import { MemoryRouter } from 'react-router-dom';
 import Sidebar from '../sidebar';
 import { defaultSiteDetails, buildSiteDetails } from './lib/fixtures';
 
+const sidebarURL = 'https://testlinkinbio.wordpress.com';
```

Comment:
Since the url displayed in the browser is no longer derived from siteSlug, but sidebarURL instead, could we update the `displays the current site url` test to reflect the new expectations?

Reply:
The test case handles the changes from siteSlug to sidebarURL. The result is the same since both variables were returning a URL.

<img width="341" alt="image" src="https://user-images.githubusercontent.com/10482592/195678285-52768bba-9789-421e-8b40-b7ef352388f0.png">

Reply:
> The result is the same since both variables were returning a URL.

siteSlug and sidebarURL might return the same value, but the expectations in the test are misleading because, ultimately, the url that's displayed in the sidebar depends on the sidebarURL prop, not siteSlug.

Maybe I'm missing something, but we're currently creating expectations that reference elements of the siteSlug ( Ex. L62 ), not the sidebarURL